src/lib/analytics/apis.native.js

```diff
@@ -1,7 +1,15 @@
 import * as SentryNative from '@sentry/react-native'
 import amplitude from 'amplitude-js'
+import analytics from '@react-native-firebase/analytics'
+
+const googleAnalyticsWrapper = {
+  push: ({ event, ...data }) => {
+    analytics().logEvent(event, data)
+  },
+}
```

Comment:
`logEvent(event, data: any = {})` looks better then `dataLayer.push(` so let's unify code with exactly that interface:

1. create `src/lib/analytics/util.js`:

```javascript
import { values, remove, isString, isNumber } from 'lodash';

export const convertToGoogleAnalytics = (data: any = {}) => {
  const values = values(data)
  const eventValues = remove(values, isNumber)
  const eventStrings = remove(values, isString)

  return {
      eventValue: eventValues.shift(),
      eventLabel: eventStrings.shift() || eventValues.shift() || JSON.stringify(values.shift()),
    }
}
```

2. 
```suggestion
import { convertToGoogleAnalytics } from './util'

class GoogleWrapper {
  constructor(analytics) {
    assign(this, { analytics })
  }
  
  logEvent(event, data: any = {}) {
    const eventData = convertToGoogleAnalytics(data)
    
    this.analytics.logEvent(event, eventData)
  }
}
```